src/tool_help.c

```diff
@@ -540,6 +540,21 @@ void tool_help(void)
   }
 }
 
+static int
+featcomp(const void *p1, const void *p2)
+{
+  /* The arguments to this function are "pointers to pointers to char", but
+     the comparison arguments are "pointers to char", hence the following cast
+     plus dereference */
+#ifdef HAVE_STRCASECMP
+  return strcasecmp(* (char * const *) p1, * (char * const *) p2);
+#elif defined(HAVE_STRCMPI)
+  return strcmpi(* (char * const *) p1, * (char * const *) p2);
+#else
+  return strcmp(* (char * const *) p1, * (char * const *) p2);
```

Comment:
Does it make sense to always use the least common denominator in order to make the output consistent across plattforms?

Reply:
Maybe. I'm not sure how common the case is without a case sensitive function here.

I think sorting it case *sensitively* makes it look funny since the lowercase features end up last.

I first wanted to use `curl_strequal` for this, but it doesn't offer the correct return code so we can't use that. (and I didn't feel like refactor everything to make such a version available)

An alternative could be to always specify features with an uppercase first letter to sort of shortcut this, but then I think "Libz" looks odd...